gym_xiangqi/piece.py

```diff
@@ -102,11 +108,18 @@ def get_actions(self, piece_id, state, actions):
         """
         Finds legal moves for the General
         """
+        if piece_id < 0:
```

Comment:
Maybe add a comment explaining this check or factoring this one line out into a function called `is_red(piece_id)` will help people understand faster?

Reply:
done